tests/postgres_tests/migrations/0002_create_test_models.py

```diff
@@ -260,4 +264,68 @@ class Migration(migrations.Migration):
             },
             bases=(models.Model,),
         ),
+        migrations.CreateModel(
+            name='Room',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('number', models.IntegerField(null=False, unique=True)),
+                ('no_people', models.IntegerField(null=False)),
+            ],
+            options={
+                'required_db_vendor': 'postgresql',
+            },
+            bases=(models.Model,),
```

Comment:
I believe you can remove `bases` as it is the default?

Reply:
Kind of indifferent to this; the existing models in the migration all have `bases=(models.Model,),`.

Reply:
Yeah, just mentioned as this was pointed out to me in the past: https://github.com/django/django/pull/11059#discussion_r266598250